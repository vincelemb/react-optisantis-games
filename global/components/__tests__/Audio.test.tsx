import React from 'react';
import { shallow } from 'enzyme';
import { Audio } from '..'

describe('<Audio />', () => {
    const props = { audioFile: 'audio-file-test.mp3', id: 'audio-test' }
    it('should render', () => {
        const wrapper = shallow(<Audio {...props} />);

        expect(wrapper).toMatchSnapshot();
    })
})