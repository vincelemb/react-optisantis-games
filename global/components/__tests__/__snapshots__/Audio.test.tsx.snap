// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Audio /> should render 1`] = `
<audio
  id="audio-test"
  src="./assets/audio/audio-file-test.mp3"
/>
`;
